src/main/java/duke/command/MalformedCommandException.java

```diff
@@ -0,0 +1,10 @@
+package duke.command;
+
+/**
+ * Signals a user command that was incorrectly formatted.
+ */
+public class MalformedCommandException extends Exception {
```

Comment:
Good name for the exception, clearly differentiates it from `UnsupportedCommandException`!